bulk-ingester/src/integration-test/java/org/alfresco/hxi_connector/bulk_ingester/processor/BulkIngestionFilterIntegrationTest.java

```diff
@@ -85,6 +87,11 @@ void shouldFilterOutNodesWithoutAllowedAspectOrWithDeniedAspect()
                 .contentInfo(new IngestEvent.ContentInfo(119625, "UTF-8", "application/pdf"))
                 .properties(parseProperties("cm:name=carp.pdf", "type=cm:content", "cm:autoVersion=true", "cm:title=", "cm:versionType=MAJOR", "cm:versionLabel=1.0", "cm:autoVersionOnUpdateProps=false", "aspectsNames=[cm:generalclassifiable, cm:versionable, cm:author, cm:thumbnailModification, cm:titled, cm:taggable, rn:renditioned, cm:auditable]", "cm:categories=fa6b38cd-442a-4f77-9d3e-dc212a6b809e", "cm:lastThumbnailModification=doclib:1708330172467", "cm:description=", "createdAt=1708330172", "cm:initialVersion=true", "createdBy=admin", "modifiedBy=admin"))
                 .build();
+        IngestEvent emailTemplate = IngestEvent.builder()
```

Comment:
How this emailTemplate node is related to allowed or denied nodes? 🤔

Reply:
It's in Data Dictionary, which is defined as denied ancestor in the filter.

Reply:
Yes, you can add this node also there to verify it for sure. I would also expect a comment here, because it wasn't obvious for me that this node is located in Data Dictionary

Reply:
Test (with empty filters) updated. Comment added.